wasatch/FeatureIdentificationDevice.py

```diff
@@ -2213,6 +2215,9 @@ def set_laser_power_perc_immediate(self, value: float):
         period_us = 1000 if self.settings.state.laser_power_high_resolution else 100
```

Comment:
I like this construct.  Probably should be using more of these since they are succinct and clear